exercise/static/exercise/assessment.js

```diff
@@ -1,4 +1,37 @@
 $(function () {
+    $.fn.addStickyButton = function (localStorageKey, actionCallback) {
```

Comment:
This line and the function definition are indented by 4 spaces, but they should only be indented one level. In JS, we have used only 2 spaces per level as you can see in the line 35:
`  $(document).on('aplus:translation-ready', function() {`

The function body is of course indented more than the outer `function(...)` line.